build/utils.py

```diff
@@ -221,8 +122,11 @@ def process_accelerate_launch_args(job_config_dict):
         )
 
     # Add training_script
-    script = os.environ.get("LAUNCH_TRAINING_SCRIPT", "/app/launch_training.py")
-    accelerate_launch_args.append(script)
+    script = os.environ.get("TRAINING_SCRIPT")
```

Comment:
This was added by @kellyaa and I don't think there is a specific need for this but allows for extensibility. You are correct, if a user sets the env var, they can run their own training script instead of the default sft_trainer one specified.

Reply:
I did that to resolve an oddity of not being able to find the script when running tests - When the docker container builds, it gets put into a different place than what is found in the structure of the project. So the tests define the location of the script in a place where it can find it.